### DISCUSSION TITLE
Ability to "Blacklist" a specific domain?

### DISCUSSION THREAD

**Discussion by ByteOtter** [Cookie-AutoDelete/Cookie-AutoDelete#1579] [Q&A / Support] [answered]
Hi,
a colleague asked me if there was a way to automatically clear the site cache and cookies of a specific domain.
He only needs that one and its subdomains as he wants to keep his login credentials and authentication tokens stored in the browser (don't ask me why).
I thought this was a possibility with CAD, I use it on my personal machine, though never configured it to clear one specific domain, and I know that whitelisting exists. Somehow I was sure "blacklisting" existed as well?
I looked around and maybe I missed something?

Sorry if this has already been asked I did not see another question like this.
Thanks for your help. :)

**Comment by kennethtran93** [comment, marked as answer]
Duplicate of #711 as a feature request, among other duplicate requests.

**Reply by ByteOtter** [reply]
Ah, sorry about that
